MaintenanceDashboard.Data/Interfaces/IPaddleContext.cs

```diff
@@ -1,14 +1,18 @@
-﻿using MaintenanceDashboard.Data.Domain;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using MaintenanceDashboard.Data.Domain;
 
-namespace MaintenanceDashboard.Data.Api
+namespace MaintenanceDashboard.Data.Interfaces
```

Comment:
warto oddzielać implementacje interfejsów od interfejsów. Tu pokusiłem się przenieść wszystkie interfejsy które potworzyłeś do innego folderu. Za dużo by było pisania czym jest interfejs i jaką role spełnia po prostu zaufaj mi, że raczej się je umieszcza w innych miejscach, W telegraficznym skrócie interfejs to kontrakt między klasami. Poprzez interfejs umawiamy się na coś, że coś zostanie wykonane. Może to ci rozświetli trochę sprawę. Jak nie, to przy przyszłych code review mogę się bardziej rozpisać jak powinniśmy działać z interfejsami